doc/index.rst

```diff
@@ -157,7 +157,44 @@ First, build the dockerfile with
   cd ros2_control_demos
   docker build . -t ros2_control_demos -f Dockerfile/Dockerfile
 
-Docker now allows us to run the demo without the GUI if configured properly. The following command runs the demo without the GUI:
+To View the Robot
+************
+
+Docker now allows us to run the demo without the GUI if configured properly. Now we can view the robot by the following procedure:
+
+After having `ros2 rolling installed on your system<https://docs.ros.org/en/rolling/Installation.html>`, we can use the joint state publisher gui package to give manual joint values to the robot. To install the package you can run:
```

Comment:
```suggestion
After having `ROS 2 installed <https://docs.ros.org/en/rolling/Installation.html>`__ on your local system (not inside the docker),  we can use ``rviz2`` to visualize the robot state and ``joint_state_publisher_gui`` package to give manual joint values to the robot. To install the package you can run:
```
Usage of the correct version of ROS 2 https://shouldiputaspacebetweenrosand2.com/ :smile: 

markup for link is with trailing `__` and a space before the `<`